department-of-veterans-affairs/caseflow
# Autosave fixes

Connects #4663 

To test:
On both the daily docket and the hearing worksheet - 
- Do not edit anything on the page and confirm that 'last saved at' is updated every 30 seconds
- Edit something on the page, confirm that 'Saving... <spinner>' shows up while saving, and then the time is updated
- Edit something on the page, update the code so that the save fails, confirm 'Save failed' shows up
